graphstorm-processing/graphstorm_processing/repartition_files.py

```diff
@@ -194,8 +197,6 @@ def create_new_relative_path_from_existing(
             "path/to/parquet-repartitioned-my-suffix/part-00003-filename.snappy.parquet"
         """
         original_relative_path_obj = Path(original_relative_path)
-        # We expect files to have a path of the form /path/to/parquet/part-00001.snappy.parquet
-        assert original_relative_path_obj.parts[-2] == "parquet"  # TODO: Remove this assumption?
```

Comment:
Do we meet any issue for it?

Reply:
This was triggered when I was using a "non standard" path to write files through pyarrow (without `parquet` in the path). I think we can remove it because it just verifies an assumption that this piece of code cannot control, it's hard for a new dev to understand why this should be here.